config/src/main/java/com/megaease/easeagent/config/ConfigFactory.java

```diff
@@ -95,6 +95,7 @@ public static GlobalConfigs loadConfigs(String pathname, ClassLoader loader) {
         // override by user special config file
         if (StringUtils.isNotEmpty(pathname)) {
             GlobalConfigs configsFromOuterFile = ConfigLoader.loadFromFile(new File(pathname));
+            LOGGER.info("Loaded user special config file: {}", pathname);
```

Comment:
If not necessary, please refrain from adding info logs to the framework.  If deemed necessary, please use debug logs.